app/views/documents/_publication_attachment.html.erb

```diff
@@ -0,0 +1,70 @@
+<%
+  extra_description ||= ""
+  published_on ||= ""
+  help_block_id = "attachment-#{attachment.id}-accessibility-help"
+  attachment_link_options = attachment.accessible? ? {} : { "aria-describedby" => help_block_id }
+  attachment_link_options['rel'] = 'external' if attachment.external?
+%>
+<%= content_tag_for(:div, attachment.becomes(Attachment), class: attachment.external? ? "hosted-externally" : "embedded") do %>
```

Comment:
Has there been a conscious decision not to make lists of attachments ordered lists?